metric-schema-markdown/src/test/java/com/palantir/metric/schema/markdown/MarkdownRendererTest.java

```diff
@@ -32,25 +32,39 @@
     @Test
     void testSimple() {
         String markdown = MarkdownRenderer.render(
-                ImmutableMap.of("com.palantir:test:1.0.0", ImmutableList.of(MetricSchema.builder()
+                "com.palantir:test", ImmutableMap.of("com.palantir:test:1.0.0", ImmutableList.of(MetricSchema.builder()
                         .namespaces("namespace", MetricNamespace.builder()
                                 .docs(Documentation.of("namespace docs"))
                                 .metrics("metric", MetricDefinition.builder()
                                         .type(MetricType.METER)
                                         .docs(Documentation.of("metric docs"))
                                         .build())
                                 .build())
+                        .namespaces("anamespace", MetricNamespace.builder()
+                                .docs(Documentation.of("namespace docs"))
+                                .metrics("metric", MetricDefinition.builder()
+                                        .type(MetricType.METER)
+                                        .docs(Documentation.of("metric docs"))
+                                        .build())
+                                .build())
                         .build())));
-        assertThat(markdown).isEqualTo("# Metrics\n\n"
-                + "## namespace\n"
+        assertThat(markdown).isEqualTo("# Metrics\n"
+                + "\n"
+                + "## `com.palantir:test:1.0.0`\n"
```

Comment:
I think I prefer the pretty printed name as the title. Could we make the full GAV part of the description to the section

Reply:
Yep, that's what the previous commit did, we can revert https://github.com/palantir/metric-schema/pull/22/commits/2fde26244e074006b5d9e3688a577b2210b975be